### ISSUE TITLE
Can you consider writting of blog or do a youtube on integrating it

### ISSUE THREAD

**Issue by chrisgate** [skoruba/IdentityServer4.Admin#48]
Dear @skoruba ,
  Thanks for this work here,
Can you consider writing a blog or do a YouTube on integrating it.

I am still thinking how to customize it and make it fit well with a new solution or existing solutions.

Thanks in anticipations.

Chris

**Comment by skoruba** [comment]
Hi @chrisgate ,

good idea, I will create some detailed post or video - step by step - how to run this solution and how to change some part of the administration.

We are working on a more suitable version that will be able to install the whole solution as a nuget package. The biggest challenge is how to distribute the UI. :)

**Comment by chrisgate** [comment]
To put this into perspective, with what we have here we wont need **https://github.com/IdentityServer/IdentityServer4.Quickstart.UI** in our project right? 

Thanks

**Comment by skoruba** [comment]
What do you mean with - wont need IDS4 Quickstart UI?

**Comment by chrisgate** [comment]
Dont mind i got it now.
but u can distribute the UI the same way identityserver4 quickstart is distributed thru powershell download.

Cheers

**Comment by skoruba** [comment]
@chrisgate You are right, I consider it because I think it is a very flexible way for future modifications such as Views/Controllers etc.

BTW: We are working on the video as you mentioned above. I will let you know. :)

Thanks!

**Comment by chrisgate** [comment]
Thanks for your team work.

We shall be waiting to see the video............cheers

**Comment by chrisgate** [comment]
any hope on this?

**Comment by skoruba** [comment]
@chrisgate Sorry for the delay, I am working on some improvements that I want to add into video tutorial as well. :) Thx

**Comment by chrisgate** [comment]
Thanks @skoruba Please mark the improvements u want to add and once u have it then add it to the video.
I said this bcoz improvements can never stop,i have seen a great framework/library that kept adding improvements and kept saying when we are done we will do videos on it,then .Net Standard came along another re-written of the framework/library and improvements stories continue and we never have the tutorial/video up till now. So all we do in that framework/library is try and error to get to solutions.

I hope u got my point.

Thanks in anticipations.

Chris Godwin.

**Comment by skoruba** [comment]
@chrisgate You are right. I got your points and I agree with them.
I will continue on creating the video for current beta version. New improvements will be part of some next video.
Thank you for your points, I really appreciated it. :)

**Comment by michshat** [comment]
@skoruba it's incredible. Thank you. Looking forward to the tutorial for the Azure.

**Comment by xmichaelx** [comment]
@michshat [Here](https://github.com/skoruba/IdentityServer4.Admin/blob/add-tutorial-for-azure-deploy/docs/Configure-Azure-Deploy.md) you have dev version of the tutorial. I'll need to go over it once more before pulling it into master.
